app/services/teacher_training_public_api/assign_temp_site_attributes.rb

```diff
@@ -26,6 +32,16 @@ def site
       end
     end
 
+    def initialize_with_generated_uuid
```

Comment:
Im wondering if we need to set a fake uuid in the permanent sync sites logic. When we add the second backfill that will be a one off to generate the `temp_site` with `uuid_generated_by_apply` set to true, after that will we ever need this again?

Reply:
We've moved this into the backfill now